Gems/Prefab/Registry/prefab_dependency_viewer.setreg

```diff
@@ -0,0 +1,12 @@
+{
+    "Amazon":
+    {
+        "Gems":
+        {
+            "PrefabDependencyViewer.Editor":
+            {
+                "AutoLoad": false
+            }
+        }
+    }
+}
```

Comment:
Still not sure what the point of this is.  Surely you'd want to turn the feature on if they turn the gem on?  The gem isn't on by default.

Reply:
The feature is a work in progress so we didn't want the customer to see the tool when they open the Editor. But I guess now the tool is going on it's feature branch we no longer need it?

Reply:
Yes. I decided to push the services that I needed in dependent service because I didn't want the application to crash if the services are not present.

If the tool is opened within the Editor(it's intended place of use), then the dependent services would guarantee that the services get activated before and destroyed after the PrefabDependencyViewer SystemComponent which is exactly what I want.